sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -694,6 +700,7 @@ export class ApplicationStudentsController extends BaseController {
       scholasticStandingChangeType: scholasticStandingChange?.changeType,
       applicationOfferingChangeRequestStatus,
       assessmentTriggerType,
+      hasFeedbackError,
```

Comment:
Should it also contain the blocking part in the name (e.g. `hasBlockingFeedbackError`)?

Reply:
Yes I was juggling back and forth with that thought and went this way. I can change it.

what about `hasBlockFundingFeedbackError`?